man/systemd.network.xml

```diff
@@ -2411,6 +2411,14 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
           then all options specified earlier are cleared. Defaults to unset.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>BindToInterface=</varname></term>
+        <listitem>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
+        to its network interface and all socket communication will be restricted to this interface.
+        Defaults to <literal>yes</literal>.
+        Disabling interface binding is mostly useful in the networks with DHCP relays.
```

Comment:
this last sentence confuses me more than clear things up. i mean, it only applies to this setup of yours where you use distinct interfaces for this relaying. maybe reword?

Reply:
Okay, I better remove that sentence at all.